### PR TITLE
Update to 1.95. JB#56339

### PR DESCRIPTION
**Pull request by pvuorela**
The license is really LGPL. Spotted only one unit test with plain GPL.

@Tomin1 @spiiroin

### CODE REVIEW

**Review by Tomin1 — APPROVED** [review]

File: rpm/libmediaart.spec [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Name:       libmediaart
 Summary:    Library for handling media art
-Version:    1.9.4
+Version:    1.9.5
 Release:    1
-License:    GPLv2
+License:    LGPLv2
```
Comment (okodron) [inline rpm/libmediaart.spec]:
```suggestion
License:    LGPLv2+
```

File: rpm/libmediaart.spec [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 URL:        https://git.gnome.org/browse/libmediaart
```
Comment (okodron) [inline rpm/libmediaart.spec:6]:
```suggestion
URL:        https://gitlab.gnome.org/GNOME/libmediaart
```

File: rpm/libmediaart.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/libmediaart.spec]: Oh yes, that seems to be even more precise.

Comment (pvuorela) [reply rpm/libmediaart.spec]: Updated.

**Review by okodron — COMMENTED** [review]
I think `BuildRequires:  gobject-introspection-devel >= 1.36` can be replaced with just `pkgconfig(gobject-introspection-1.0)`.

**Comment by okodron** [comment]
Also `with-docs` build option has been replaced with `gtk_doc`: https://gitlab.gnome.org/GNOME/libmediaart/-/blob/1.9.5/meson_options.txt#L3

**Comment by pvuorela** [comment]
> Also `with-docs` build option has been replaced with `gtk_doc`: https://gitlab.gnome.org/GNOME/libmediaart/-/blob/1.9.5/meson_options.txt#L3

Indeed. Default false so it built fine but think we can now just omit the whole thing.

**Pull request merged**